.evergreen/config.yml

```diff
@@ -2389,9 +2389,9 @@ axes:
           SKIP_ECS_AUTH_TEST: true
       # TODO(BUILD-17329): Update this to Ubuntu 22 after we add a new ECS task
```

Comment:
Should we remove this comment, or do we still plan on bumping to Ubuntu 22? If so, do we need to update this comment with a different build ticket?

Reply:
Removed, Ubuntu 22 only supports MongoDB 6.0+.